instrumentation/apache-nifi/apache-nifi-1.24.0/library/src/main/java/io/opentelemetry/instrumentation/nifi/v1_24_0/model/StandardIdentifier.java

```diff
@@ -0,0 +1,205 @@
+      }
+      catch (ClassCastException e) {
+
+          System.out.println(">>>>>> DEBUG : PROBLEM !! In Connectable block : " + e.getMessage());
+      }
+    }
+
+    public String getUuid() {
+        return uuid;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Long getSessionId() {
+        return sessionId;
+    }
+
+    public static Long getSessionId(StandardProcessSession standardProcessSession) throws Exception {
+        return (Long) FieldUtils.readField(standardProcessSession, "sessionId", true);
+    }
+
+    public boolean isMultiSessionProcessor() {
+        return isMultiSessionProcessor;
+    }
+
+    public static boolean isMultiSessionProcessor(StandardProcessSession standardProcessSession) throws Exception {
+
+        Connectable connectable = getConnectable(standardProcessSession);
+        return isBinFileProcessor(connectable);
+    }
+
+    public Connectable getConnectable() {
+        return connectable;
+    }
+
+    public static Connectable getConnectable(StandardProcessSession standardProcessSession) throws Exception {
+
+        RepositoryContext repositoryContext = getRepositoryContext(standardProcessSession);
+        return repositoryContext.getConnectable();
+    }
+
+    public ConnectableType getConnectableType() {
+        return connectableType;
+    }
+
+    public ComponentSessionType getComponentSessionType() {
+        return componentSessionType;
+    }
+
+    public static ComponentSessionType getComponentSessionType(StandardProcessSession standardProcessSession) throws Exception {
+
+        if(isMultiSessionProcessor(standardProcessSession)) {
+            return ComponentSessionType.MULTI_SESSION;
+        }
+        else {
+            return ComponentSessionType.SINGLE_SESSION;
+        }
+    }
+
+    public StandardProcessSession getStandardProcessSession() {
+        return standardProcessSession;
+    }
+
+    public static boolean isProcessorConnectableType(StandardProcessSession standardProcessSession) throws Exception {
+
+        Connectable connectable = getConnectable(standardProcessSession);
+        return connectable.getConnectableType().equals(ConnectableType.PROCESSOR);
+    }
+
+    public static boolean isBinFileProcessor(Connectable connectable) {
+
+        if (connectable.getConnectableType().equals(ConnectableType.PROCESSOR)) {
+
+           Object object = connectable.getRunnableComponent();
+
+           if(object instanceof Processor) {
+
+               Processor processor = (Processor) object;
+               System.out.println(">>>>>> DEBUG IN-BINFILES-METHOD : " + processor.getIdentifier());
+
+               //String x = org.apache.nifi.processor.util.bin.BinFiles.class.getName();
+               
+               // IF UNCOMMENT TWO LINES BELOW, MISSING CLASS OCCURS
+               //String x = org.apache.nifi.processors.standard.LogAttribute.class.getName();
```

Comment:
Need community's help in understanding why uncommenting these lines causes a class not found error when we include it in the build.gradle.kts